extensions/amp-story/1.0/amp-story.js

```diff
@@ -1443,39 +1452,45 @@ export class AmpStory extends AMP.BaseElement {
     const uiState = this.getUIType_();
     this.storeService_.dispatch(Action.TOGGLE_UI, uiState);
 
-    if (uiState !== UIType.MOBILE || this.isLandscapeSupported_()) {
+    const isLandscape = this.isLandscape_();
+    const isLandscapeSupported = this.isLandscapeSupported_();
+    this.storeService_.dispatch(
+        Action.TOGGLE_LANDSCAPE, isLandscapeSupported && isLandscape);
```

Comment:
I'm okay with this, but as a larger point, this is the third place that really could benefit from storing the amp-story-page width/height in the store.  I think this would be more understandable if it just read `dispatch(TOGGLE_LANDSCAPE, page.width > page.height)`; the way it's written here takes a little bit of mental processing (and an understanding of our UITypes and API surface) to understand that this is based on the page size and not the viewport size.

I don't want to block this more, but if you agree, can you add a `// TODO(#20832)` to base this on the size of the amp-story-page?

Reply:
That's actually a good point, thanks. :)
I removed the `LANDSCAPE_STATE` from the store, it's no longer needed with the direction we're taking.

Reply:
Can you add this `TODO`, though?  This implementation is still based on `isLandscapeSupported && isLandscape`, which I think is suboptimal (just from a readability perspective)

Reply:
Done